Source/Scene/ParticleSystem.js

```diff
@@ -0,0 +1,295 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/Cartesian2',
+        '../Core/Cartesian3',
+        '../Core/Event',
+        '../Core/Matrix4',
+        '../Core/Math',
+        '../Core/JulianDate',
+        '../Core/Color',
+        './BillboardCollection',
+        './Particle',
+        './CircleEmitter'
+    ], function(
+        defaultValue,
+        defined,
+        Cartesian2,
+        Cartesian3,
+        Event,
+        Matrix4,
+        CesiumMath,
+        JulianDate,
+        Color,
+        BillboardCollection,
+        Particle,
+        CircleEmitter) {
+    "use strict";
+
+    var ParticleSystem = function(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        this.particles = defaultValue(options.particles, []);
+        this.forces = defaultValue(options.forces, []);
+
+        this.emitter = defaultValue(options.emitter, new CircleEmitter({radius: 0.5}));
+
+        this.modelMatrix = Matrix4.clone(defaultValue(options.modelMatrix, Matrix4.IDENTITY));
+        this.emitterModelMatrix = Matrix4.clone(defaultValue(options.emitterModelMatrix, Matrix4.IDENTITY));
+
+        this.startColor = defaultValue(options.startColor, Color.clone(Color.WHITE));
+        this.endColor = defaultValue(options.endColor, Color.clone(Color.WHITE));
+
+        this.startScale = defaultValue(options.startScale, 1.0);
+        this.endScale = defaultValue(options.endScale, 1.0);
+
+        this.rate = defaultValue(options.rate, 5);
+        this.bursts = defaultValue(options.bursts, null);
```

Comment:
I made this change along with a few others in acd31e71824a8f0bd75a07432a51091c01d5ef28 that removed the unnecessary use of defaultValue.